prompt2model_demo.py

```diff
@@ -181,8 +181,37 @@ def main():
             TaskType.TEXT_GENERATION, status["instruction"], status["examples"]
         )
         line_print("Retrieving dataset...")
+        line_print("Do you want to retrieve dataset automatically? (y/n)")
+        auto_retrieve = False
+        while True:
+            line = input()
+            if line.lower() == "y":
+                auto_retrieve = True
+                break
+            elif line.lower() == "n":
+                auto_retrieve = False
+                break
+            else:
+                line_print("Invalid input. Please enter y or n.")
+
         retriever = DescriptionDatasetRetriever()
-        retrieved_dataset_dict = retriever.retrieve_dataset_dict(prompt_spec)
+
+        if auto_retrieve:
+            while True:
+                line_print("Enter the number of data points you want to transform:")
```

Comment:
Might be good to mention here what will happen to the remaining data points (e.g. that they will be discarded from the dataset).